### DISCUSSION TITLE
Check for convergence in *stochastic* lambda

### DISCUSSION THREAD

**Discussion by Aariq** [levisc8/ipmr#40] [Ideas]
Right now it seems like it doesn't make sense to use `conv_plot()` or `is_conv_to_asymptotic()` on stochastic IPMs since both functions use the lambdas for each transition year.  I'm not super familiar with the literature on this, but it makes intuitive sense to me that you'd want to check for convergence in the *stochastic* lambda.  Couldn't you do this by checking for convergence after calculating a cumulative mean on all the lambdas?

**Comment by Aariq** [comment]
Just browsing through code from Ellner, Childs and Rees and they also check for convergence in stochastic lambda using a cumulative mean.
```
#how fast do they converge?

Ls.N <- cumsum(rt.N)/(1:n.est)
Ls.V <- cumsum(rt.V)/(1:n.est)

n.samp <- n.est

plot(Ls.N[1:n.samp],type="l")
points(Ls.V[1:n.samp],type="l",col="red")
abline(h=mean(rt.N))
```
Where `rt.N` = log(sum(nt1)/sum(nt)), which is the average annual growth rate, `rt.V` is an approximation of total reproductive value, and `n.est` is just the number of iterations.

**Reply by levisc8** [reply]
This has been addressed by #44, right?

**Reply by Aariq** [reply]
Yeah, this has been dealt with now.
